pool/pool_aio_test.go

```diff
@@ -136,8 +136,6 @@ func TestPoolAio(t *testing.T) {
 		image := fmt.Sprintf("%s:%s", version.image, version.version)
 
 		t.Run(image, func(t *testing.T) {
-			t.Parallel()
```

Comment:
why is it removed in that commit?

Reply:
because it is a part of updating GH workflow. Runner fails with parallel docker test executing.